README.md

```diff
@@ -35,11 +35,11 @@ val appComponent = AppComponent::class.create()
 val repo = appComponent.repo
 ```
 
-## Download 
+## Gradle 
 
 Using [ksp](https://github.com/google/ksp)
 
-`settings.gradle`
+`settings.gradle.kts`
```

Comment:
[Kotlin DSL is the default now](https://blog.gradle.org/kotlin-dsl-is-now-the-default-for-new-gradle-builds) so I thought this might be appropriate.